readme_renderer/clean.py

```diff
@@ -41,9 +41,7 @@
 
     # Custom Additions
     "*": ["id"],
-    "hr": ["class"],
```

Comment:
I wasn't able to find out why `hr` was expected to contain a class. It was added at the same time when `hr` was authorized as a tag itself, by @dstufft in #21 . If you happen to remember what it was about, I'm interested. But I couldn't find a relevant `hr.<somthing>` rule in Warehouse CSS.